ui/shared/IconButton.qml

```diff
@@ -4,26 +4,33 @@ import QtQuick.Layouts 1.3
 import Qt.labs.platform 1.1
 import "../imports"
 
-Rectangle {
-    signal clicked
+RoundButton {
     property int iconWidth: 14
     property int iconHeight: 14
-    property alias icon: imgIcon
+    property alias iconImg: imgIcon
     property bool clickable: true
     property string iconName: "plusSign"
+    property color color: Style.current.blue
+
+    icon.width: iconWidth
+    icon.height: iconHeight
 
     id: btnAddContainer
```

Comment:
Not a big deal, but that `id` is not really representative. I guess it was just kept ad part of the old AddButton.

Reply:
That's correct, I just didn't touch that